### PR TITLE
examples: add missing v.mod file

### PR DESCRIPTION
**Pull request by JalonSolov**
Adds missing v.mod file to actually make the example work correctly.

### CODE REVIEW

File: examples/submodule/v.mod [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Module {
+	name: 'submodule'
+	description: ''
```
Comment (vincenzopalazzo) [inline examples/submodule/v.mod:3]:
```suggestion
	description: 'and example of sumbodule'
```

File: examples/submodule/v.mod [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+	version: ''
```
Comment (vincenzopalazzo) [inline examples/submodule/v.mod:4]:
```suggestion
	version: '0.0.1'
```

File: examples/submodule/v.mod [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+	license: ''
```
Comment (vincenzopalazzo) [inline examples/submodule/v.mod:5]:
```suggestion
	license: 'MIT'
```

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
A good example attract good user (maybe)

**Comment by JalonSolov** [comment]
Just FYI - the `v.mod` file was created by running `v init`.  If you have suggestions for how it creates the file, that would be a different PR.

**Pull request merged**